cmd/etcdbackup/etcdbackup.go

```diff
@@ -24,6 +25,7 @@ var (
 	blobName    = flag.String("blobname", "", "Name of the blob (without the container)")
 	destination = flag.String("destination", "", "Where to place the blob on the filesystem")
 	maxBackups  = flag.Int("maxbackups", 6, "Maximum number of backups to keep")
+	timeout     = flag.Duration("timeout", 1*time.Hour, "Timeout of the request to the OpenShift resource provider.")
```

Comment:
This can probably be hardcoded since it's always going to be a single request that we know how much it takes more or less.

Reply:
I would keep it consistent with other cmd files. Makes easier to understand. each cmd, which calls fakeRP would have this and it is set via flags.